README.md

```diff
@@ -1,33 +1,35 @@
-# Data Converters for TEKTELIC Network Server
-This repository contains the data converters that are to be used on TEKTELIC Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms (such as AWS, TTN, etc).
+# Data Converters for TEKTELIC Devices
+This repository contains the data converters that are to be used on TEKTELIC & TTN Network Server for TEKTELIC Sensors. These data converters can be used as a reference for other platforms.
```

Comment:
Note that the Tektelic V2 data converters conform to the LoRa Alliance® LoRaWAN® Payload Codec API.

Reply:
Should I mention the version such as  LoRa Alliance® LoRaWAN® Payload Codec API 1.x.x?